assets/app/components/site/Pages/pagesContainer.js

```diff
@@ -15,10 +16,6 @@ class Pages extends React.Component {
     super(props);
   }
 
-  componentDidMount() {
-    fetchFiles(this.props.site, this.props.params);
```

Comment:
Do we no longer need to do this fetch the first time we mount the component? _starts digging around..._